### PR TITLE
+pro enable validatePullRequest locally

### PR DESCRIPTION
**Pull request by ktoso**
Resolves https://github.com/akka/akka/issues/19569

### CODE REVIEW

**Comment by 2m** [comment]
Nice. LGTM.

File: project/ValidatePullRequest.scala [hunk 1]
```diff
@@ -134,18 +149,34 @@ object ValidatePullRequest extends AutoPlugin {
 
       val prId = (sourceBranch in ValidatePR).value
 
+      val target = (targetBranch in ValidatePR).value
```
Comment (rkuhn) [inline project/ValidatePullRequest.scala]: Could you add a paragraph to CONTRIBUTING that explains how to use this feature, especially against a different branch?

**Comment by rkuhn** [comment]
Great feature @ktoso, I want to start using it right away! :-)

**Comment by drewhk** [comment]
LGTM.

**Comment by ktoso** [comment]
Failure was ssh problems (unrelated).
Adding info to CONTRIBUTING.md and merging

**Comment by ktoso** [comment]
Added guide to CONTRIBUTING

**Comment by ktoso** [comment]
FYI @sirthias @jrudolph we don't have the `stream-and-http` meta project anymore, using the `validatePullRequest` is now the suggested way of testing "everything my code could have impacted".
We can work on making it smarter still ofc.

**Pull request merged**